cypress/integration/pages/index.spec.ts

```diff
@@ -23,10 +23,6 @@ context('/ on macbook-13', () => {
     cy.findByTestId('BlocksList.viewAllBlocksLink').should('have.attr', 'href', '/blocks')
   })
 
-  it('should have BlocksList.viewAllBlocksButton', () => {
-    cy.findByTestId('BlocksList.viewAllBlocksButton').should('have.attr', 'href', '/blocks')
-  })
-
```

Comment:
If the test is failing don't just remove them. Anw this is failing because there are 2 instances of "BlocksList.viewAllBlocksButton" you can give another data-testid for the one in the mobile view.